Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:6
+WORKDIR /usr/src/app
+RUN npm install
+COPY . .
```

Comment:
Is this how Docker works? I would assume you're telling it to clone the working directory to the working directory, effectively doing nothing, am I wrong?